source/backend/dal/Repositories/AgreementRepository.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Pims.Core.Extensions;
+using Pims.Dal.Entities;
+using Pims.Dal.Helpers.Extensions;
+
+namespace Pims.Dal.Repositories
+{
+    /// <summary>
+    /// Provides a repository to interact with agreements within the datasource.
+    /// </summary>
+    public class AgreementRepository : BaseRepository<PimsAcquisitionChecklistItem>, IAgreementRepository
+    {
+        #region Constructors
+
+        /// <summary>
+        /// Creates a new instance of a AgreementRepository, and initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="dbContext"></param>
+        /// <param name="user"></param>
+        /// <param name="logger"></param>
+        public AgreementRepository(PimsContext dbContext, ClaimsPrincipal user, ILogger<AgreementRepository> logger)
+            : base(dbContext, user, logger)
+        {
+        }
+
+        #endregion
+
+        #region Methods
+
+        public IEnumerable<PimsAcqChklstItemType> GetAllChecklistItemTypes()
```

Comment:
is this used anywhere? also, is this repository a good place for this?

Reply:
bad copypasta